### DISCUSSION TITLE
FortiClient VPN

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#16] [General]
# FortiClient VPN

Check if FortiClient VPN is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/954ee80d-1f14-4be2-ac15-b8c595b96b59

**Comment by Alex590263** [comment]
Try on my surface pro x, last insider update, with forticlient from the microsoft store
I managed to connect to the VPN however no reception bytes. Works perfectly on my PC in x86_64

**Reply by talynone** [reply]
Try this work-around https://community.fortinet.com/t5/FortiClient/Technical-Tip-FortiClient-for-ARM-Architecture/ta-p/248361

**Reply by Alex590263** [reply]
Already tested, I use the same method and its only works on my 1335U, on the surface pro x it doesn't want to know anything
The connection is established but no reception bit

**Reply by dagasoft** [reply]
I have the same problem. VPN with microsoft store Forticlient negociates connection, but no inboud traffic. Sometimes it works, but most of them no.
